src/components/GradBackground.tsx

```diff
@@ -0,0 +1,17 @@
+import { FC } from 'react'
+import gradBG from '../../public/gradBG.svg'
+
+
```

Comment:
If you did ` npm run format` don't you catch these empty lines

Reply:
Hey @NsdHSO you proposed this as a question, but it doesn't needed to be. When reviewing, try to make the feedback actionable. We can discussions in the comments or discord.